#include "Exception.hpp"

int Exception::getNumber()
{
    return number;
}

std::string Exception::getError()
{
    return error;
}